index.html

```diff
@@ -209,8 +209,10 @@ <h4>Service Three</h4>
 
             </div><!-- / .row -->
             <!-- Submit -->
-            <input type="submit" value="View More Services" class="btn" style="float: right; position: relative;
-            left: -40%; text-align: left;">
+            <form action="services.html">
+              <input type="submit" value="View More Services" class="btn" style="float: right; position: relative;
+              left: -40%; text-align: left;">
+            </form>
```

Comment:
Use `<a><button></button></a>` instead of a form, since there's no inputs needed